docs/documentation/images_in_documentation_files.md

```diff
@@ -7,82 +7,26 @@ title: Images in Documentation Files
 
 Images in documentation markdown files are divided into 2 different types:
 - Images that appear in integration/script/playbook readmes. These images only appear in https://xsoar.pan.dev/. They do not appear in the Cortex XSOAR/XSIAM product UI.
-- Images that appear in pack readmes and integration description files. These images appear in both https://xsoar.pan.dev/ and in the Cortex XSOAR/XSIAM product UI. 
+- Images that appear in pack readmes and integration description files. These images appear in both https://xsoar.pan.dev/ and in the Cortex XSOAR/XSIAM product UI.
 
-## Integration/Script/Playbook Readme Images
+## Rules for uploading images to markdown files:
 
-When creating markdown `README` documents for playbooks, integrations, or scripts that appear in https://xsoar.pan.dev/ only, you can use a relative or absolute URL.
+When creating markdown documentation files, you should use a relative path to the `doc_files` folder located in the pack's root path.

```

Comment:
give example

Reply:
There's an example several lines below in line 20.